data-prepper-plugins/common/build.gradle

```diff
@@ -14,6 +14,7 @@ plugins {
 }
 dependencies {
     api project(':data-prepper-api')
+    implementation project(':data-prepper-plugins:blocking-buffer')
```

Comment:
I think you only need this for the tests. It can be made to be `testImplementation` and should sit with the other `testImplementation` dependencies.